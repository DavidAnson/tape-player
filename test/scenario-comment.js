// @ts-check

"use strict";

const tape = require("tape");
require("../index.js");

tape("comment test", (test) => {
  test.plan(2);
  test.pass();
  test.comment("comment message");
  test.pass();
});
