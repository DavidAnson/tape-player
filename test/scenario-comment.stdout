comment test

Tests:    1
Asserts:  2
Failures: 0
